cmd/test.go

```diff
@@ -229,6 +239,15 @@ func (t *Test) GetCmd() *cobra.Command {
 				}
 			}
 
+			if buildDelay <= time.Duration(30)*time.Second {
+				fmt.Printf("Warning: buildDelay is set to %d, incase your docker container takes more time to build use --buildDelay to set custom delay\n", buildDelay)
+				if isDockerCmd {
+					fmt.Println("Example usage:\n", `keploy test -c "docker run -p 8080:808 --network myNetworkName myApplicationImageName" --buildDelay 35s\n`, "\nor\n", `keploy test -c "docker run -p 8080:808 --network myNetworkName myApplicationImageName" --buildDelay 1m\n`)
+				} else {
```

Comment:
Only mention test command usage here.

Reply:
Sorry for the confusion, what does 'Only mention test command usage here. ' mean , Shall I  remove the warning message 🤔

Reply:
Sorry, I didn't notice. You don't need to remove the command or any warning message. Just fix the port forwarding.